codice/ddf
# DDF-5676 Fix Solr docker and itests on Windows

#### What does this PR do?
Fixes the dictionary file being too large on Windows due to additional CR characters after checking out from git.

- The jute.maxbuffer system property was not properly set for Solr or Zookeeper.
- Additional CRs caused the dictionary to be too large on Windows and fail with the default max buffer value.
- Line endings were also incorrect for sh or bat scripts depending on the operating system used to build the distributions.
- Zookeeper 3.4.x does not trim CRs from config files when using the sh script.

#### Who is reviewing it? 

@bakejeyner 

#### Select relevant component teams: 

@codice/solr 

#### Ask 2 committers to review/merge the PR and tag them here.

@stustison
@rzwiefel
@kcwire 

#### How should this be tested?

- Verify `distribution/test/itests/test-itests-ddf` works on Windows.
- Verify `distribution/docker/solrcloud` works on Windows.

#### Any background context you want to provide?

#### What are the relevant tickets?
Fixes: #5753 

#### Screenshots


#### Checklist:

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.
